pkg/installer/mdsd/util.go

```diff
@@ -0,0 +1,212 @@
+package mdsd
+
+import (
+	"bytes"
+	"html/template"
+	"io"
+	"path"
+	"path/filepath"
+	"strings"
+
+	ignutil "github.com/coreos/ignition/v2/config/util"
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
+	"github.com/openshift/installer/pkg/asset/ignition"
+)
+
+func AddStorageFiles(config *igntypes.Config, base string, uri string, templateData interface{}) (err error) {
```

Comment:
Given that there are exactly 5 files involved here, we could avoid copying this code from the installer and take the same approach as in #227 - embed all the files individually and hard-code the destination path.

Reply:
I'll be using this function for the other 2 PRs as well. Probably need to add this to the dnsmasq function too which I can fix in another PR

Reply:
Maybe create a separate package for the data like e.g. pkg/installer/data/bootstrap then.